samples/slack/slack_settings

```diff
@@ -1,2 +1 @@
-TOKEN=yourtoken
-SLACK_WEBHOOK_URL=https://yoursite.slack.com/services/hooks/incoming-webhook?token=$TOKEN
\ No newline at end of file
+SLACK_WEBHOOK_URL=https://hooks.slack.com/services/xxx/yyy/zzz
```

Comment:
Will see about testing this out soon. Until then, this file needs an ending newline. Thanks for this!